Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/PDFScrollingPresentationController.mm

```diff
@@ -201,7 +201,7 @@
         auto destinationRect = pageBoundsRect;
         destinationRect.scale(documentLayout.scale());
 
-        auto pageContainerLayer = [&](PDFDocumentLayout::PageIndex pageIndex) {
+        SUPPRESS_UNCOUNTED_LAMBDA_CAPTURE auto pageContainerLayer = [&](PDFDocumentLayout::PageIndex pageIndex) {
```

Comment:
Could you file a radar & add the link here? You can file it in WebKit Tools and assign it to me.

Reply:
Done: rdar://144977686